client/state/data-layer/wpcom/me/settings/test/index.js

```diff
@@ -145,15 +145,15 @@ describe( 'wpcom-api', () => {
 		} );
 
 		describe( '#userSettingsSaveSuccess', () => {
-			test( 'should dispatch user settings update and clear all unsaved settings on full save', () => {
+			test( 'should dispatch user settings update and clear all unsaved settings on full save', async () => {
 				const dispatch = jest.fn();
 				const action = { type: 'DUMMY' };
 
-				userSettingsSaveSuccess( action, {
+				await userSettingsSaveSuccess( action, {
 					language: 'qix',
 				} )( dispatch );
 
-				expect( dispatch ).toHaveBeenCalledTimes( 3 );
+				expect( dispatch ).toHaveBeenCalledTimes( 4 );
```

Comment:
As this patch shows, the number of calls is an implementation detail that doesn't need to be tested.

Reply:
Couldn't agree more. It's very rarely useful. I decided to keep it here, but we can reconsider those at any time in the future.